node/src/WebRtcTransport.ts

```diff
@@ -12,24 +12,34 @@ import {
 import { WebRtcServer } from './WebRtcServer';
 import { SctpParameters, NumSctpStreams } from './SctpParameters';
 
-export type WebRtcTransportOptions =
+export interface WebRtcTransportListenIndividual
 {
-	/**
-	 * Instance of WebRtcServer. Mandatory unless listenIps is given.
-	 */
-	webRtcServer?: WebRtcServer;
 	/**
 	 * Listening IP address or addresses in order of preference (first one is the
 	 * preferred one). Mandatory unless webRtcServer is given.
 	 */
-	listenIps?: (TransportListenIp | string)[];
+	listenIps: (TransportListenIp | string)[];
```

Comment:
This is now mandatory but it shouldn't (if webrtcserver is given).

Reply:
It was incomplete indeed, now it works properly, please check updated version.